rules-tests/TypeDeclaration/Rector/Property/CompleteVarDocTypePropertyRector/Fixture/recursive_multiple_class_string_array.php.inc

```diff
@@ -41,7 +41,7 @@ class FactoryC {}
 final class RecursiveMultipleClassStringArray
 {
     /**
-     * @var array<string, mixed[]>
+     * @var array<string, array<class-string<\Rector\Tests\TypeDeclaration\Rector\Property\CompleteVarDocTypePropertyRector\Fixture\FactoryA>|class-string<\Rector\Tests\TypeDeclaration\Rector\Property\CompleteVarDocTypePropertyRector\Fixture\FactoryB>|class-string<\Rector\Tests\TypeDeclaration\Rector\Property\CompleteVarDocTypePropertyRector\Fixture\FactoryC>|class-string<\Rector\Tests\TypeDeclaration\Rector\Property\CompleteVarDocTypePropertyRector\Fixture\InvokableA>|class-string<\Rector\Tests\TypeDeclaration\Rector\Property\CompleteVarDocTypePropertyRector\Fixture\InvokableB>|class-string<\Rector\Tests\TypeDeclaration\Rector\Property\CompleteVarDocTypePropertyRector\Fixture\InvokableC>>>
```

Comment:
@TomasVotruba I think previous `mixed[]` is correct, this change make too long type, original PR for it at https://github.com/rectorphp/rector-src/pull/520

Reply:
Oh crap, I've missed this. Could you check it? The change was only in removing `UnionTypeFactory`

Reply:
I will try.